packages/app/src/components/cms/inline-choropleth.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  ChoroplethConfiguration,
+  DataScope,
+  DataScopeKey,
+  MetricKeys,
+} from '@corona-dashboard/common';
+import { get, isString } from 'lodash';
+import useSWRImmutable from 'swr/immutable';
+import { isDefined } from 'ts-is-present';
+import { useIntl } from '~/intl';
+import { SiteText } from '~/locale';
+import { useReverseRouter } from '~/utils/use-reverse-router';
+import { DynamicChoropleth } from '../choropleth';
+import { ErrorBoundary } from '../error-boundary';
+import { Metadata } from '../metadata';
+import { InlineLoader } from './inline-loader';
+import { getDataUrl } from './logic/get-data-url';
+
+interface InlineChoroplethProps {
+  title: string;
+  configuration: ChoroplethConfiguration<DataScopeKey, MetricKeys<DataScope>>;
+}
+
+export function InlineChoropleth(props: InlineChoroplethProps) {
+  const { configuration } = props;
+
+  const { siteText } = useIntl();
+
+  const dateUrl = getDataUrl(undefined, undefined, configuration, 'choropleth');
+
+  const { data } = useSWRImmutable(dateUrl, (url: string) =>
+    fetch(url).then((_) => _.json())
+  );
+
+  if (!isDefined(data)) {
+    return <InlineLoader />;
+  }
+
+  const dataOptions = {
+    getLink: isDefined(configuration.link)
+      ? get(useReverseRouter, configuration.link, undefined)
+      : undefined,
+    tooltipVariables: parseTooltipVariables(
+      configuration.tooltipVariables,
+      siteText
+    ),
+  };
```

Comment:
Links are not working right now because it's trying to get the links from the reverseRouter hook itself instead of from the router it returns.

```suggestion
  const { siteText } = useIntl();
  
  const reverseRouter = useReverseRouter();

  const dateUrl = getDataUrl(undefined, undefined, configuration, 'choropleth');

  const { data } = useSWRImmutable(dateUrl, (url: string) =>
    fetch(url).then((_) => _.json())
  );

  if (!isDefined(data)) {
    return <InlineLoader />;
  }

  const dataOptions = {
    getLink: isDefined(configuration.link)
      ? get(reverseRouter, configuration.link, undefined)
      : undefined,
    tooltipVariables: parseTooltipVariables(
      configuration.tooltipVariables,
      siteText
    ),
  };
```

Reply:
haha, good catch, this was a dumb mistake of mine haha